src/main/java/com/github/andriiyan/sprongtraining/impl/utils/DumpUtils.java

```diff
@@ -48,14 +50,14 @@ public DumpUtils(@NonNull String rootFolder, @NonNull FileUtils fileUtils, int i
      * File's extensions will variate depending on the chose Serializer (which injects into the FileUtils). possible
      * variants of extensions are: .json, .object.
      */
-    public DumpResult dump() {
+    DumpResult dump() {
         try {
             final Collection<User> users = dumpUsers();
-            logger.info("Users " + users.toString() + " were dumped into the " + rootFolder + "/users" + suffix());
+            logger.debug("Users " + users.toString() + " were dumped into the " + rootFolder + "/users" + suffix());
```

Comment:
BTW, I didnt said 'Replace all INFO logs with DEBUG'. I just asked to choose wisely)